src/nvim/diff.c

```diff
@@ -1425,11 +1425,11 @@ void diff_clear(tabpage_T *tp)
 /// @param lnum
 ///
 /// @return diff status.
-int diff_check(win_T *wp, linenr_T lnum)
+linenr_T diff_check(win_T *wp, linenr_T lnum)
```

Comment:
I think `int` would be more appropriate here since the result may not be related to a line number.